aws/resource_aws_db_instance.go

```diff
@@ -1816,3 +1928,31 @@ var resourceAwsDbInstanceUpdatePendingStates = []string{
 	"storage-full",
 	"upgrading",
 }
+
+func expandRestoreToPointInTime(l []interface{}) *rds.RestoreDBInstanceToPointInTimeInput {
+	if len(l) == 0 || l[0] == nil {
+		return nil
+	}
+	tfMap, ok := l[0].(map[string]interface{})
+	if !ok {
+		return nil
+	}
+	input := &rds.RestoreDBInstanceToPointInTimeInput{}
```

Comment:
Nit: Between groups of statements, I like a line.
```suggestion

	input := &rds.RestoreDBInstanceToPointInTimeInput{}
```